validators/terravaloper1f337thg2cxu6yny9hquec42p4s2a27awyu8cgf/profile.json

```diff
@@ -0,0 +1,13 @@
+{
+  "contact": {
+    "email": "contact@lunanauts.net",
+    "twitter": "@Lunanauts_Lunc"
+  },
+    "notifications": { 
+        "email": "contact@lunanauts.net" 
+    }
+

```

Comment:
@lunanauts

Seems that you mistakenly deleted the closing bracket `}` and added some newlines that aren't needed.  Thanks!